specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabricManagedClusters/preview/2021-01-01-preview/examples/ManagedClusterVersionsListByOS_example.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "resRg",
+    "location": "eastus",
+    "osType": "Windows",
+    "api-version": "2021-01-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": [
+        {
```

Comment:
doesn't look like a well formatted resource here

Reply:
missing things like id, type, names, etc.

Reply:
I removed this path. We added it by mistake this is not supported yet and it's going to be different.